concepts/external-crates/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+Most programs have dependencies on some libraries. Managing those by hand can be a pain. Luckily, the Rust ecosystem comes standard with `cargo`! `cargo` can manage dependencies for a project.
+
+You will often see external packages being referred to as "crates" in Rust. A crate is a compilation unit in Rust. A crate can be compiled into a binary or into a library.
+
+Most of the time, adding an external dependency is as simple as adding a line to your `Cargo.toml` file.
+
+In the corresponding exercise, such an external dependency was added for you.
```

Comment:
```suggestion
```

Concept docs should stand alone, and there isn't a 1:1 mapping between concepts and concept exercises, so we shouldn't say this.